src/Driver.cpp

```diff
@@ -876,3 +921,33 @@ int Driver::getChannelNumber(void)
     sendCommand(command);
     return waitResponseInt(command);
 }
+
+// Set System Time for current time
+void Driver::setSystemTimeNow(void)
+{
+    string command = "AT!UT";
+    double time_now = base::Time::now().toSeconds();
+    cout << "time_now "<< time_now <<endl;
```

Comment:
Don't add this kind of debugging output in libraries. If you want to use loggging, use [Rock's logger](http://rock-robotics.org/master/documentation/debugging/index.html)